.eslintrc.js

```diff
@@ -34,6 +34,9 @@ module.exports = {
 
     // warn us if we're using @ts-ignore etc..
     '@typescript-eslint/ban-ts-comment': 'warn',
+
+    // Old code uses any a lot, just show a warn instead
+    '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
I think this is already 'warn':

<img width="694" alt="Screenshot 2022-02-15 at 12 29 50" src="https://user-images.githubusercontent.com/93216813/154062382-2f96008c-4dbb-4dc0-a36d-d781821b8f5c.png">

Could you try reinstalling your node modules?

Reply:
It could be that my IDE was using a cached TS server. Does it hurt to have it explicitly in config?

Reply:
Nope doesn't matter